cluster-scope/overlays/ocp-prod/oauths/cluster_patch.yaml

```diff
@@ -16,7 +16,23 @@ spec:
           - preferred_username
         clientID: moc-openshift-ocp-prod
         clientSecret:
-          name: moc-sso-client-secret
+          name: clientsecret-moc
         extraScopes: []
         issuer: https://sso.massopen.cloud/auth/realms/moc
       type: OpenID
+    - mappingMethod: claim
+      name: moc-testing
+      openID:
+        claims:
+          email:
+            - email
+          name:
+            - name
+          preferredUsername:
+            - preferred_username
+        clientID: moc-openshift-ocp-prod
```

Comment:
wouldn't it be a different client id?

Reply:
Nope. The client id is effectively a "username", which is unique per realm. We have to realms here, so we use the same name in both (like I can be "larsks" on github, and also on twitter).